### PR TITLE
feat(crons): Simplified monitor creation experience

### PR DESCRIPTION
**Pull request by davidenwang**
Simpler flow for creation, eventually will have a quickstart section at the bottom to help with onboarding.

Before:
image

After:
image
image

### CODE REVIEW

File: static/app/views/monitors/components/monitorForm.tsx [hunk 1]
```diff
@@ -1,7 +1,10 @@
-import {Fragment, useRef} from 'react';
+import {useRef} from 'react';
 import styled from '@emotion/styled';
 import {Observer} from 'mobx-react';
 
+import Alert from 'sentry/components/alert';
+import {RadioField} from 'sentry/components/forms';
```
Comment (evanpurkhiser) [inline static/app/views/monitors/components/monitorForm.tsx:6]: nit: can you import this directly instead of from the barrel class

Comment (davidenwang) [reply static/app/views/monitors/components/monitorForm.tsx:6]: TIL its called a barrel class

Comment (evanpurkhiser) [reply static/app/views/monitors/components/monitorForm.tsx:6]: sorry module, but yeah barrel module

File: static/app/views/monitors/components/monitorForm.tsx [hunk 2]
```diff
@@ -202,130 +221,142 @@ function MonitorForm({
...
+  padding: 0;
+`;
+
+const StyledTextField = styled(TextField)`
+  padding: 0;
+`;
+
+const StyledSentryProjectSelectorField = styled(SentryProjectSelectorField)`
+  padding: 0;
+`;
```
Comment (evanpurkhiser) [inline static/app/views/monitors/components/monitorForm.tsx:336]: Unfortunate lol

Comment (davidenwang) [reply static/app/views/monitors/components/monitorForm.tsx:336]: yea this is the worst LOL

File: static/app/views/monitors/components/monitorForm.tsx [hunk 3]
```diff
@@ -202,130 +221,142 @@ function MonitorForm({
...
+
+const StyledListItem = styled(ListItem)`
```
Comment (evanpurkhiser) [inline static/app/views/monitors/components/monitorForm.tsx:338]: Do we have to change this? How do we do it in alerts?

Comment (davidenwang) [reply static/app/views/monitors/components/monitorForm.tsx:338]: copied from alerts haha

Comment (davidenwang) [reply static/app/views/monitors/components/monitorForm.tsx:338]: https://github.com/getsentry/sentry/blob/952e4de9a068694babaf795c2949fedbdd1ba606/static/app/views/alerts/rules/metric/ruleConditionsForm.tsx#L564

Comment (evanpurkhiser) [reply static/app/views/monitors/components/monitorForm.tsx:338]: @scttcper should we extract this out or something?

**Review by evanpurkhiser — APPROVED** [review]

**Pull request merged**